README.md

```diff
@@ -229,9 +235,9 @@ class Counter extends ReComponent {
 
 ### Manage State Across the Tree
 
-You often want to pass state properties down to other children in order for them to behave properly. Some times, however, this tree is very deep and it might be inefficient to go through the whole tree (and thereby updating it) to pass a value down.
+Often times we want to pass state properties to descendants that are too far deep in the application tree. In order to do so the components in between need to pass those properties to their respective children until we reach the desidered component. This pattern is usually called [prop drilling](https://blog.kentcdodds.com/prop-drilling-bb62e02cb691).
```

Comment:
I don't want to praise context as a solution to prop passing (because I think that this usually is not a big deal) but want to propose it as a solution to avoid all interim layers to re-render to avoid performance problems. How do you like this instead:

> Often times we want to pass state properties to descendants that are very deep in the application tree. In order to do so, the components in between need to pass those properties to their respective children until we reach the desired component. This pattern is usually called [prop drilling](https://blog.kentcdodds.com/prop-drilling-bb62e02cb691) and usually what you want.
> 
> Sometimes, however, the layers in-between are expensive to re-render causing your application to become janky. Fortunately, React 16.3.0 introduced a new API called [`createContext()`](https://reactjs.org/docs/context.html#reactcreatecontext) that we can use to solve this issue by using context to pass those properties directly to the target component and skipping the update of all intermediate layers:

Reply:
Sounds good!

> called prop drilling and usually what you want.

called prop drilling and it is usually what you want to do.